.eslintrc.json

```diff
@@ -0,0 +1,22 @@
+{
+  "extends": "airbnb-base",
+  "parserOptions": {
+      "ecmaVersion": 6,
+      "sourceType": "module"
+  },
+  "env": {
+    "mocha": true
+  },
+  "rules": {
+    "no-param-reassign": "WARN" ,
+    "no-underscore-dangle": "WARN",
+    "prefer-destructuring": ["ERROR", {"object": true, "array": false}],
+    "no-unused-vars": [
+      "error",
+      {
+        "varsIgnorePattern": "should|expect"
+      }
+    ],
+    "no-unused-expressions": "off"
```

Comment:
Dėl kokios priežasties šita taisyklė išjungta?

Reply:
Testuose buvo tokių išraiškų kaip expect(foo).to.be.true arba expect(foo).to.be.empty. Tada pamaniau išjungti taisyklę. Tačiau vėliau tiesiog pabandžiau pakeisti išraišką į expect(foo).to.be.equal(true)  ir viskas buvo ok, tik pamiršau pašalinti taisyklės išjungimą.